### ISSUE TITLE
Multiple not required branches show up!

### ISSUE THREAD

**Issue by secrashi** [edamontology/edam-browser#128]
On clicking "only data", "only format", "only operations" etc from the drop-down menu, multiple branches show up, as shown in the figure attached:

![WhatsApp Image 2021-04-16 at 5 18 36 PM (1)](https://user-images.githubusercontent.com/56053777/115020529-57e9d980-9ed8-11eb-8481-18ffdee74e57.jpeg)

If @bryan-brancotte @matuskalas @hmenager approve, then I'll change this to show up the direct branches only, as shown in the 'after' pic above!

**Comment by raashika03** [comment]
But then there'll be no use of cookies which is enabled to store the state of branches, either closed or open. @secrashi

**Comment by bryan-brancotte** [comment]
Hi @secrashi 

I did not get your point, could you rephrase it ?

**Comment by secrashi** [comment]
Hi @bryan-brancotte, I meant that when we click on a topic say `only operation` from the drop-down menu, then various branches expand as shown in figure below: 

![WhatsApp Image 2021-04-27 at 10 10 14 AM](https://user-images.githubusercontent.com/56053777/116186284-ef54f500-a740-11eb-87b8-65813d8b102f.jpeg)

Instead, only direct branches to operations must show up! 

> But then there'll be no use of cookies which is enabled to store the state of branches, either closed or open. @secrashi

@raashika03, I'm proposing to change the state when the branches first show up, so the cookies will function the same way. Hope that answers your doubt!

**Comment by bryan-brancotte** [comment]
Thanks for your clarification @secrashi, the current behavior is that we save per-branch last seen term, and overall last seen term, so when opening a topic in the global tree, then selecting only operation, we try to open the last seen term, but as it is a topic we fall back on the last seen term when looking at operation which in you screenshot is `Sequence comparison`.

So the behavior is expected, but the question of "Is it what we want" can be raised.

**Comment by secrashi** [comment]
Ohh, ohhk, Now it makes more sense, I was aware of storing last seen term and cookies stuff, but I wasn't aware of Sequence comparison. Thank you so much for letting me know!


> So the behavior is expected, but the question of "Is it what we want" can be raised.

I currently don't have any better method than this, but I would like to work on this if you have something better in your mind!

**Comment by bryan-brancotte** [comment]
I don't have sorry, feel free to close this issue and move to another topic.
